blueprint/scripts/subcommands.go

```diff
@@ -84,5 +94,47 @@ func SubCommands() []cli.Command {
 				},
 			},
 		},
+		{
+			Name:   "add-label",
+			Usage:  "This action assign a single label from a single labelable resource",
+			Action: cmd.LabelAdd,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "id",
+					Usage: "Script Id",
+				},
+				cli.StringFlag{
+					Name:  "label",
+					Usage: "Label name",
+				},
+				cli.StringFlag{
+					Name:   "resource_type",
+					Usage:  "Resource Type",
+					Value:  "script",
+					Hidden: true,
+				},
+			},
+		},
+		{
+			Name:   "remove-label",
+			Usage:  "This action de-assign a single label from a single labelable resource",
```

Comment:
I would use as verb here either remove or unassign, and it also needs an s: `This action unassigns [...]`